app/views/foreman_tasks/tasks/index.html.erb

```diff
@@ -2,20 +2,7 @@
 <% title_actions csv_link, help_button %>
 <% stylesheet 'foreman_tasks/tasks' %>
 
-<script>
-
-var currentTwoPaneTask;
-
-$(document).on('click', ".table-two-pane td.two-pane-link", function(e) {
-  currentTwoPaneTask = $(this).find("a");
```

Comment:
I found out that this var is used in the show template. With removing it, this https://github.com/theforeman/foreman-tasks/blob/master/app/views/foreman_tasks/tasks/show.html.erb#L10-L17 becomes dead code.